docs/sidebars.js

```diff
@@ -59,7 +59,8 @@ module.exports = {
             'maps/adding-points-and-lines',
             'maps/latlon',
             'maps/map-series',
-            'maps/tilemap-series'
+            'maps/tilemap-series',
+            'maps/mapBubble-series'
```

Comment:
The map series first, the other series type next sorted alphabetically (A to Z).